design/run_and_collect_data.md

```diff
@@ -0,0 +1,27 @@
+#run.py proposal
```

Comment:
Check that the folder isn't being blocked by the `.gitignore` file. The `git add :/` command should add everything both new and updated in the working tree